index.css

```diff
@@ -87,6 +95,219 @@ nav {
   }
 
   .social-icon:hover {
-    width: 25px;
+    color: #396df2;
+  }
+
+  .works-sec {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .work-card {
+    display: grid;
+    width: 327px;
+    height: 552px;
+    padding-left: 7%;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    align-items: center;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    margin-bottom: 88px;
+    background: #fff;
+  }
+
+  .lng-wrk-card {
+    height: 596px;
+  }
+
+  .work-image-cont {
+    width: 295px;
+    height: 220px;
+    background-color: pink;
+    border-radius: 8px;
+  }
+
+  .work-list {
+    display: flex;
+  }
+
+  .work-list-item {
+    float: left;
+    margin-left: 10px;
+    align-items: center;
+  }
+
+  .canopy {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 14px;
+    color: #344563;
+    padding-top: 5px;
+  }
+
+  .project-title {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    margin-top: 5px;
+    margin-bottom: 0;
+  }
+
+  .project-desc {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    color: #344563;
+    width: 295px;
+    height: 76px;
+  }
+
+  .project-cont {
+    width: 90%;
+    align-items: flex-start;
+  }
+
+  .project-cont ul {
+    display: block;
+    list-style: none;
+    margin-left: -40px;
+  }
+
+  .project-cont ul li {
+    float: left;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    background: #ebebff;
+    border-radius: 8px;
+    align-items: center;
+    padding: 4px 12px;
+    color: #6070ff;
+    margin-right: 8px;
+  }
+
+  .project-cont ul li:hover {
+    background-color: blue;
+    color: white;
+  }
+
+  .see-pjct-btn {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    height: 48px;
+    left: 0;
+    width: 124px;
```

Comment:
- [ ] You may consider removing this width on your button. since a button of that size on a mobile screen of 375px is a bit too much :smile: